File: pimotor.py
import os
import RPi.GPIO as GPIO
import serial
import time

ser = serial.Serial('/dev/ttyAMA0',9600,timeout=1)

while(True):
	pipein = open("/var/www/FIFO_pimotor", 'r')
	line = pipein.readline()
	if line == "forward":
		ser.write('1')
	elif line == "backwards":
		ser.write('2')
	elif line == "right":
		ser.write('3')
	elif line == "left":
		ser.write('4')
	elif line == "stop":
		ser.write('0')
	elif line == "fire":
		ser.write('5')

	pipein.close()

File: pimotor.js

var ajax_pimotor;

var direction = "stop"; 
 
if(window.XMLHttpRequest) {
  ajax_pimotor = new XMLHttpRequest();
  //The XMLHttpRequest object is used to exchange data with a server.
}
else {
  ajax_pimotor = new ActiveXObject("Microsoft.XMLHTTP");
  //For IE
}
 
function pi_movement() {
  ajax_pimotor.open("GET","pimotor.php?movement=" + direction, true);
  //open(method,url,async)
  ajax_pimotor.send();
}
 
function movement_left() {
  direction = "left";
  pi_movement();
}
 
function movement_right() {
  direction = "right";
  pi_movement();
}
 
function movement_forward() {
  
  direction = "forward";
  pi_movement();
}
 
function movement_backward() {
  direction = "backwards";
  pi_movement();
}
 
function movement_stop() {
  direction = "stop";
  pi_movement();
}

function movement_fire(){
  direction = "fire";
  pi_movment();
}

function movement_reload(){
  direction = "reload";
  pi_movment();
}